src/common/util/feature-flags.js

```diff
@@ -12,7 +12,10 @@ const bucketMap = {
   stn: [FEATURE_NAMES.sessionTrace],
   err: [FEATURE_NAMES.jserrors, FEATURE_NAMES.metrics],
   ins: [FEATURE_NAMES.pageAction],
-  spa: [FEATURE_NAMES.spa]
+  spa: [FEATURE_NAMES.spa],
+  sr: [FEATURE_NAMES.sessionReplay],
+  srf: [FEATURE_NAMES.sessionReplay],
+  sre: [FEATURE_NAMES.sessionReplay]
```

Comment:
Should probably add comments on how these three features are mutually exclusive and what each one "means".

Reply:
I think these flags are going to change / maybe go away, in waiting mode there

Reply:
should be just `sr`, adding `feat-srf` and `feat-sre` overly complicates

Reply:
agreed